README.md

```diff
@@ -152,21 +152,9 @@ Tests can be run using [npm][npm]:
 
 ## Deployment
 
-When new code is pushed to master Travis CI runs a test suite against it. If those test
-pass [fabric][fabric] is used to deploy the new version to the server running
-under https://wo-ist-markt.de.
+Deployment is automatically done with Netlify. See the [contribution guide](CONTRIBUTING.md) for more details.
```

Comment:
I would not duplicate the information in the _README.md_ and _CONTRIBUTING.md_. The second sentence should be enough - or we fully remove the "Deployment" section in the _README.md_. What do you think?